ffs/arbitration.md

```diff
@@ -0,0 +1,114 @@
+
+## 8. Conduct of the Arbitration
+
+The tribunal will make every effort to conduct the arbitration in a timely manner. The tribunal has the discretion to adopt procedural measures as they consider appropriate and to determine the admissibility, relevance, materiality and weight of any evidence. The tribunal shall make all of its decisions by majority vote.
+
+## 9. Publication of Decision
+
+Once a decision has been reached, a plain text document explaining the decision shall be cryptographically signed with the use of [Unique Ring Signature](https://github.com/monero-project/urs) keyring. The text of the decision and the signature hash shall be posted in the issue opened for the dispute. In the event of a split decision (2/1), arbitrators are prohibited from disclosing who was the dissenting arbitrator.
+
+## 10. Proof
+
+If a party makes a claim, it is their responsibility to support it. A screenshot is reasonable evidence if it is not manipulated. Giving details related to an event is plausible evidence if the details can be independently confirmed. Payments can be proved with the submission of Transaction IDs, Transaction Keys, Payment IDs, public address, blockchain hash and block height data. Verifying a signed message with a private key can prove ownership of a public address. Cryptographically timestamping can prove information existed at a given point in time. The tribunal may examine metadata embedded in documents and electronic communications in order to assess the evidential value of such information.
+
+## 11. Conflicts of Interest
+
+A conflict of interest arises if a party has one of the following relationships to an arbitrator:
+
+- Business partner
+- Employer or employee
+- Stakeholder
+- Close friend or relative
+
+All parties must reveal any evidence of a conflict of interest as soon as possible before or during a dispute. In assessing conflict of interests, arbitrators shall have regard for the [IBA Guidelines on Conflicts of Interest](https://www.ibanet.org/Document/Default.aspx?DocumentUid=E2FE5E72-EB14-4BBA-B10D-D33DAFEE8918).
+
+## 12. Status
+
+There is no centralized authority, entity, or institution that controls, dictates, regulates, or makes decisions regarding the relationship between the parties and the arbitrators. The arbitrators are volunteers and parties directly interact with the arbitrators by virtue of an arbitration agreement and these Rules. Nothing contained in the Rules create an employer and employee, partnership, joint venture, or any other fiduciary relationship. The arbitrators are not agents for, or act on behalf of, any of the parties.
+
+## 13. Place of Arbitration
+
+The place of arbitration shall be Geneva, Switzerland. The award shall be deemed to have been made at the place of arbitration. The tribunal may conduct the arbitration at any location they consider appropriate, including, online through videoconferencing or any other electronic communication mediums.
```

Comment:
>The place of arbitration shall be Geneva, Switzerland. 

😕 

Does this really apply to an internet arbitration in which you've previously said `The tribunal is explicitly authorized by the parties to resolve any dispute, *ex aequo et bono*, in accordance with what is considered to be fair and equitable without regard to any applicable law.`?

Reply:
Yes. A place of arbitration should be indicated even if the process is conducted online. Switzerland is a pro-arbitration jurisdiction and is party to the [New York Convention](http://www.newyorkconvention.org), which would make the award more easily enforceable in many [contracting states](http://www.newyorkconvention.org/countries).

The reference to applicable law relate to the rules that apply to the performance of the obligations under the contract, while the place of arbitration deals with the rules that relate to the process of arbitration and the recognition and enforcement of awards.